inc/project.class.php

```diff
@@ -1205,10 +1207,12 @@ static function showShort($id, $options = []) {
          $first_col = '';
          $color     = '';
          if ($item->fields["projectstates_id"]) {
-            $query = "SELECT `color`
-                      FROM `glpi_projectstates`
-                      WHERE `id` = '".$item->fields["projectstates_id"]."'";
-            foreach ($DB->request($query) as $color) {
+            $iterator = $DB->request([
+               'SELECT' => 'color',
+               'FROM'   => 'glpi_projectstates',
+               'WHERE'  => ['id' => $item->fields['projectstates_id']]
+            ]);
+            while ($color = $iterator->next()) {
```

Comment:
You have conflict between `$color` used as loop variable and `$color` variable set on next line.